xlators/mgmt/glusterd/src/glusterd-volgen.c

```diff
@@ -91,7 +91,6 @@ xlator_instantiate_va(const char *type, const char *format, va_list arg)
     char *volname = NULL;
     int ret = 0;
     xlator_t *this = THIS;
```

Comment:
nit: 'this' is ONLY used in error log messages. So in the positive flow, there's no point of even calling 'THIS'. I'd argue that functions such as this, should not be bothering with this unconditional assignment.

(However, let's do it in a different patch...)